ofl/bodonimoda/METADATA.pb

```diff
@@ -0,0 +1,31 @@
+name: "Bodoni Moda"
+designer: "Owen Earl"
+license: "OFL"
+category: "SERIF"
+date_added: "2020-09-28"
+fonts {
+  name: "Bodoni Moda"
+  style: "normal"
+  weight: 400
+  filename: "BodoniModa[opsz,wght].ttf"
+  post_script_name: "BodoniModa-Regular"
+  full_name: "Bodoni Moda Regular"
+  copyright: "Copyright 2020 The Bodoni Moda Project Authors (https://github.com/indestructible-type)"
+}
+fonts {
+  name: "Bodoni Moda"
+  style: "italic"
+  weight: 400
+  filename: "BodoniModa-Italic[opsz,wght].ttf"
+  post_script_name: "BodoniModa11-Italic"
+  full_name: "Bodoni Moda Italic"
+  copyright: "Copyright 2020 The Bodoni Moda Project Authors (https:github.comindestructible-type)"
```

Comment:
I updated the metadata file which shows that the regular is good, while the italic still have the wrong postscript name and a typo in the copyright URL (https://)